tests/console/commands/test_publish.py

```diff
@@ -103,3 +111,21 @@ def test_publish_dry_run(
     assert "Publishing simple-project (1.2.3) to PyPI" in output
     assert "- Uploading simple-project-1.2.3.tar.gz" in error
     assert "- Uploading simple_project-1.2.3-py2.py3-none-any.whl" in error
+
+
+def test_skip_existing_output(
+    app_tester: ApplicationTester, http: type[httpretty.httpretty]
+):
+
+    http.register_uri(
+        http.POST, "https://upload.pypi.org/legacy/", status=409, body="Conflict"
+    )
+
+    exit_code = app_tester.execute(
+        "publish --skip-existing --username foo --password bar"
+    )
+
+    assert exit_code == 0
+
+    output = app_tester.io.fetch_output()
+    assert "Uploading simple-project-1.2.3.tar.gz File exists. Skipping" in output
```

Comment:
```suggestion
    error = app_tester.io.fetch_error()
    assert "- Uploading simple-project-1.2.3.tar.gz File exists. Skipping" in output
```

Check error instead of out.